listenbrainz/webserver/templates/user/import.html

```diff
@@ -58,4 +58,13 @@ <h3>Import a Last.fm backup file</h3>
       return false;
     }
   </script>
+  <script type="text/javascript">
+    function check() {
+      var i = document.getElementById("lastfm_input");
```

Comment:
Our pages have jquery installed on them and this can be done a lot simpler with jquery. Can you please change this code to use jquery?

Reply:
Ok, I'll modify it :+1:

Reply:
I did some research on jquery code for it and found that it is nearly of same length as this one.  Should we keep this or change it then?

Reply:
I think that he refers to `$("#lastfm_input")` as being shorter to get the input box. I agree that if we use jquery in other places that we should also use it here. Additionally, in this case I don't think it's necessary to assign it to the variable `i` - the name is not very descriptive and it's not used in multiple places. You could do this entire check inside the if block.

Reply:
similarly for setting the disabled attribute